src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String path) {
+        try (BufferedWriter bw = new BufferedWriter(new java.io.FileWriter(path))) {

```

Comment:
It's not recommended to use fully qualified name for FileWriter. Consider importing the java.io.FileWriter class at the beginning of your file. You can rename your class `FileWriter` to for example `FileWriterService` so it will not be the same as the java class `java.io.FileWriter`.